diff.py

```diff
@@ -1406,7 +1526,6 @@ def _normalize_bl(self, mnemonic: str, row: str) -> str:
         row, _ = split_off_address(row)
         return row + "<ignore>"
 
-
```

Comment:
(this newline will get added back the next time someone runs `black`)

Reply:
Reverted and re-ran with `black`